### DISCUSSION TITLE
[Feature request] Row Dragging - Grid to Grid - to external dropzone

### DISCUSSION THREAD

**Discussion by MidoAhmed** [Comcast/react-data-grid#3758] [General]
I would to request a new feature Row Dragging :
- Grid to Grid - 
- To external dropzone 

if possible to consider this feature and add this in your backlog if it's not already done
Thanks in advance.

**Comment by amanmahajan7** [comment]
We won't be adding this feature to RDG. I believe it may be possible to add a custom solution using `renderRow/renderCell`.
